src/Libraries/DSOffice/Excel.cs

```diff
@@ -416,7 +417,7 @@ internal object[][] Data
         /// </summary>
         /// <param name="wbook"></param>
         /// <param name="sheetName"></param>
-        internal WorkSheet (WorkBook wbook, string sheetName)
+        internal WorkSheet (WorkBook wbook, string sheetName, bool overWrite = true)
```

Comment:
For some reason I don't see `Worksheet` node in Dynamo... but if these documentation ever face the customer, then please make sure to complete them (including those existing ones). Thanks!